experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -44,9 +44,10 @@ parsed_args["nh_poly"] = 4
 
 import ClimaCoupler
 pkg_dir = pkgdir(ClimaCoupler)
-coupler_output_dir = joinpath(pkg_dir, "experiments/AMIP/moist_mpi_earth")
+COUPLER_OUTPUT_DIR = joinpath(pkg_dir, "experiments/AMIP/moist_mpi_earth/output", mode_name)
+run(`mkdir -p $COUPLER_OUTPUT_DIR`)
```

Comment:
```suggestion
COUPLER_OUTPUT_DIR = joinpath(pkg_dir, "experiments/AMIP/moist_mpi_earth/output", mode_name)
mkpath(COUPLER_OUTPUT_DIR)
```
I think this should work, too? This is probably simpler

Reply:
replaced with `isdir(COUPLER_OUTPUT_DIR*"_artifacts") || mkdir(COUPLER_OUTPUT_DIR*"_artifacts")`